week-1/interpret/time-format.js

```diff
@@ -15,15 +12,38 @@ console.log(result);
 
 // a) How many variable declarations are there in this program?
 
+//* 7 const movieLength = 8784; // length of movie in seconds
+// const remainingSeconds = movieLength % 60;
+// const totalMinutes = (movieLength - remainingSeconds) / 60;
+// const remainingMinutes = totalMinutes % 60;
+// const totalHours = (totalMinutes - remainingMinutes) / 60;
+// const remainingHours = totalHours % 24;
+// const result = `${remainingHours}:${remainingMinutes}:${remainingSeconds}`;
+
+
 // b) How many function calls are there?
 
+//* const totalMinutes = (movieLength - remainingSeconds) / 60;
```

Comment:
Neither of these are function calls I'm afraid. The brackets here are just being used for mathamatical reasons. There is one function call in the code

Reply:
Sorry Anna, I am not sure which is that "one function" ?

Reply:
Hi,
Sorry for the misunderstanding,  I'll explain again .This question asks you to identify functions in the code. You answered with two snippets of code:
```
//* const totalMinutes = (movieLength - remainingSeconds) / 60;
// const totalHours = (totalMinutes - remainingMinutes) / 60;
```
Unfortunately, neither of these snippets contain a function call. They do contain brackets, but they are not function calls. These are being used the same way brackets are used in maths.

There is a function called in the code. Can you find it?

Reply:
Is it just "console.log(result);" ?